rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java

```diff
@@ -487,7 +487,7 @@ private String eth_getStorageAt(HexAddressParam address, HexNumberParam storageI
                     .getStorageValue(addr, DataWord.valueOf(HexUtils.strHexOrStrNumberToByteArray(storageIdx.getHexNumber())));
 
             if (sv == null) {
-                s = "0x0";
+                s = "0x" + String.join("", Collections.nCopies(64, "0"));
```

Comment:
just an idea, could it simply be in the form of the string literal (probably defined as a constant) - same as in the tests, like this? - `0x0000000000000000000000000000000000000000000000000000000000000000`

Reply:
Yeah, maybe that makes it more readable. I'll update it.